run-pre-commit/action.yml

```
@@ -50,6 +56,20 @@ runs:
 
         echo "$LOCATION_DIR" | tee -a "$GITHUB_PATH"
 
+    - name: Abort nix-github-token input is unset
+      if: ${{ inputs.nix && !inputs.nix-github-token }}
```

FYI, the interpolation isn't required.

```suggestion
      if: inputs.nix && !inputs.nix-github-token
```